bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,178 @@
+					}
+
+					// Update file content.
+					updatePluginHeader( {
+						key: moduleDir,
+						slug: pluginSlug,
+						version: pluginVersion,
+						pluginModulePath: modulePath,
+					} );
+
+					// Update text domain.
+					updateTextDomain( {
+						slug: pluginSlug,
+					} );
+				} catch ( error ) {
+					log(
+						formats.error(
+							`${ error }`
+						)
+					);
+				}
+			}
+		} catch ( jsonError ) {
+			log(
+				formats.error(
+					`Error parsing JSON string: "${ jsonError }"`
+				)
+			);
+		}
+	} );
+};
+
+/**
+ * Updates the `load.php` file header information.
+ *
+ * @param {[]Settings} settings Plugin settings.
+ */
+async function updatePluginHeader( settings ) {
+	const { key, version, slug, pluginModulePath } = settings;
+	const loadFile = path.join( '.', 'load.php' );
+	const loadFileContent = fs.readFileSync( loadFile, 'utf-8' );
+	const buildLoadFile = path.join( '.', 'build/' + slug + '/load.php' );
+	const buildLoadFileContent = fs.readFileSync( buildLoadFile, 'utf-8' );
+
+	const regex = /\/\\*\\*[\s\S]+?(?=\*\/)/mi;
+	const pluginComment = loadFileContent.match( regex )?.[ 0 ];
+	const newPluginComment = buildLoadFileContent.match( regex )?.[ 0 ];
+
+	fs.writeFileSync( buildLoadFile, buildLoadFileContent.replace( newPluginComment, pluginComment ) );
+
+	let updatedBuildLoadFileContent = fs.readFileSync( buildLoadFile, 'utf-8' );
+
+	const moduleFilePattern = path.join( pluginModulePath, 'load.php' );
+	const moduleFiles = path.resolve( '.', moduleFilePattern );
+	const { name, description } = await getModuleDataFromHeaders( moduleFiles );
+
+	// Get module header data.
+	const moduleData = {
+		pluginname: name,
+		plugindescription: description,
+		pluginuri: '/tree/trunk/modules/' + key,
+		pluginversion: version,
+		textdomain: slug,
+	};
+
+	// Map of module header => object property.
+	const headers = {
+		'Plugin Name': 'pluginname',
+		'Plugin URI': 'pluginuri',
+		Description: 'plugindescription',
+		Version: 'pluginversion',
+		'Text Domain': 'textdomain',
+	};
+
+	const headerRegex = new RegExp( `(?<header>${ Object.keys( headers ).join( '|' ) }): (?<value>(.*?)+)`, 'gmi' );
+	const matches = [ ...updatedBuildLoadFileContent.matchAll( headerRegex ) ];
+
+	matches.forEach( ( match ) => {
+		const { header, value } = match.groups;
+		const prop = headers[ header ];
+		let moduleDataValue = moduleData[ prop ];
+
+		if ( value && prop && moduleDataValue ) {
+			if ( prop === 'pluginuri' ) {
+				moduleDataValue = value + moduleDataValue;
+			}
+			fs.writeFileSync( buildLoadFile, updatedBuildLoadFileContent.replace( new RegExp( value ), moduleDataValue ) );
+			updatedBuildLoadFileContent = fs.readFileSync( buildLoadFile, 'utf-8' );
+		}
+	} );
+}
+
+/**
+ * Updates the text domain in the build folder files content.
+ *
+ * @param {[]Settings} settings Plugin settings.
+ */
+async function updateTextDomain( settings ) {
+	const patterns = [
+		path.resolve( __dirname, '../../../build/**/*.php' ),
```

Comment:
This is incorrect as it matches all built plugins and their files. You should rather pass the individual built plugin's path to this function and use that here.

Something like this:
```suggestion
		path.resolve( pluginPath, './**/*.php' ),
```